src/app/zap-templates/zcl/operational-credentials-cluster.xml

```diff
@@ -81,8 +81,7 @@ fabric-scoped data.</description>
 
     <command source="client" code="0x06" name="AddOpCert" optional="false">
       <description>Sender is requesting to add the new node operational certificate.</description>
-      <arg name="NOC" type="OCTET_STRING"/>
-      <arg name="ICACertificate" type="OCTET_STRING"/>
+      <arg name="OperationalCert" type="OCTET_STRING" array="true"/>
```

Comment:
Why `array="true"` here?  This is a single OCTET_STRING, not an array of them, right?

Reply:
It's expected to be an array of certs. Not sure if `array="true"` is actually helping with anything here.

Reply:
The other places I see `array="true"` in our XML are things like:
```
<arg name="protocolsSupported" type="OTADownloadProtocol" array="true"/>
```
which in the spec has type `list[DownloadProtocolEnum]`.  And the codegen for that generates `uint8_t *`, not `uint8_t`....

Reply:
Removed the `array="true"`

Reply:
Per offline discussion: this should not have `array="true"` and `array="true"` is kinda broken for OCTET_STRING...